code/modules/projectiles/projectile/special/plasma.dm

```diff
@@ -32,18 +22,18 @@
 			return -1
 
 /obj/item/projectile/plasma/adv
-	damage = 28
+	damage = 7
 	range = 5
 	mine_range = 5
 
 /obj/item/projectile/plasma/adv/mech
-	damage = 40
+	damage = 10

```

Comment:
This shouldn't be nerfed, there are no alternatives for mecha, and isn't spammable.